### PR TITLE
Remove cowsay from CentOS 8 Stream container

### PR DESCRIPTION
**Pull request by felixfontein**
Having cowsay active makes it an annoying container for CI :)

### CODE REVIEW

**Comment by dmsimard** [comment]
The idea I had was for cowsay to ensure these are, in fact, /not/ to be used for production.

We could try to think of something else that happens during runtime -- because few actually read the warnings/disclaimers in the READMEs/docs.

Maybe just a single echo via cowsay that says this is not intended for production use ?

**Comment by felixfontein** [comment]
Ref: #10

**Comment by dmsimard** [comment]
Ok, let's do #10 -- between that and the images being purposefully called ``test-image`` and ``test-ee``, hopefully that is sufficient.

File: ansible-test/centos-stream8/build.sh [hunk 1]
```diff
@@ -11,10 +11,6 @@ buildah run "${build}" -- /bin/bash -c "dnf update -y && dnf install --alloweras
 # Extra python dependencies
 buildah run --volume ${SCRIPT_DIR}:/tmp/src:z "${build}" -- /bin/bash -c "pip3 install -r /tmp/src/requirements.txt"
 
-# Cows are a good reminder that these are not for production use :)
-buildah run "${build}" -- /bin/bash -c "dnf -y install epel-release && dnf -y install cowsay && dnf -y remove epel-release && dnf clean all"
-buildah config --env ANSIBLE_NOCOWS=0 "${build}"
```
Comment (dmsimard) [inline ansible-test/centos-stream8/build.sh:16]:
```
 ____
< :( >
 ----
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
```

**Comment by felixfontein** [comment]
Thanks for reviewing and merging!

**Pull request merged**